config/initializers/sentry.rb

```diff
@@ -0,0 +1,4 @@
+Raven.configure do |config|
+  config.dsn = ENV.fetch('SENTRY_DSN', nil)
+  config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
+end
```

Comment:
Questions:
1. why not putting the DSN in the encrypted credentials?
2. if I remember correctly sentry had a key, which is taken from the ENV I presume? Why not using the encrypted credentials for that?

Reply:
This sounds like the same question (DSN/key are the same thing). I've moved these to credentials in the staging env PR so there's not an encrypted file conflict :+1: